awsagentprovider/src/test/java/software/amazon/opentelemetry/javaagent/providers/AwsMetricAttributeGeneratorTest.java

```diff
@@ -461,6 +462,21 @@ public void testRemoteAttributesCombinations() {
     // Validate behaviour of various combinations of DB attributes, then remove them.
     validateAndRemoveRemoteAttributes(DB_SYSTEM, "DB system", DB_OPERATION, "DB operation");
 
+    // Validate db.operation not exist, but db.statement exist, where SpanAttributes.DB_STATEMENT is
```

Comment:
Can we also add a test case where both `db.operation` and `db.statement` exists but we pick the value from the former?

Reply:
Added test case 9.